### PR TITLE
New Collection Card Design

### PR DESCRIPTION
**Pull request by alchemistgo87**
Addresses: #648

### CODE REVIEW

File: src/components/CollectionPreviewCard.tsx [hunk 1]
```diff
@@ -17,9 +17,9 @@ interface CollectionPreviewCardProps {
 
 interface CollectionPreviewSkeletonProps {
   background: JSX.Element;
-  preview: JSX.Element;
+  // preview: JSX.Element;
   name: JSX.Element;
-  size: JSX.Element | boolean;
+  dropTime: JSX.Element | boolean;
```
Comment (kristianeboe) [inline src/components/CollectionPreviewCard.tsx:22]: How can this be boolean? :)

File: src/components/CollectionPreviewCard.tsx [hunk 2]
```diff
@@ -33,32 +33,32 @@ function CollectionPreviewSkeleton(props: CollectionPreviewSkeletonProps): JSX.E
         props.className
       )}
     >
-      <div className="relative flex aspect-[306/440] w-full">{props.background}</div>
+      <div className="relative flex aspect-[1/1] w-full">{props.background}</div>
```
Comment (kristianeboe) [inline src/components/CollectionPreviewCard.tsx:36]: Could this be aspect-square ?

File: src/components/CollectionPreviewCard.tsx [hunk 3]
```diff
@@ -72,10 +72,10 @@ export function CollectionPreviewLoadingCard(): JSX.Element {
     <CollectionPreviewSkeleton
       className="animate-pulse"
       background={<div className="h-full w-full bg-gray-900 bg-opacity-50" />}
-      preview={<div className="h-full w-full rounded bg-gray-800" />}
+      // preview={<div className="h-full w-full rounded bg-gray-800" />}
       name={<div className="h-16 w-full rounded bg-gray-800" />}
-      size={<div className="h-8 w-[35%] rounded bg-gray-800" />}
-      floor={<div className="h-8 w-[35%] rounded bg-gray-800" />}
+      dropTime={<div className="h-11 w-[50%] rounded bg-gray-800" />}
```
Comment (kristianeboe) [inline src/components/CollectionPreviewCard.tsx:77]: w-[50%] could be w-1/2

**Review by kristianeboe — APPROVED** [review]
Please take a look at these comments before merging

File: src/components/CollectionPreviewCard.tsx (hunk 1 above)
Comment (alchemistgo87) [reply src/components/CollectionPreviewCard.tsx:22]: I just changed the name, data and type will be updated with update in graphql data.

**Pull request merged**